### DISCUSSION TITLE
Spacy MLFlow logger not Logging parameters or metrics

### DISCUSSION THREAD

**Discussion by dscsoler** [explosion/spaCy#12931] [Help: Coding & Implementations] [answered]
I'm trying an spacy spancat model, and I'm trying to log the results on the command line I'm using ML flow, but I the results don't show on the website. This is my code on the conf file: 
     
     [training.logger]
     @loggers ="spacy.MLflowLogger.v1"

This is a python script I'm using to run the code:

        import mlflow
        import subprocess

        mlflow.set_tracking_uri("http://mlflow......")
        experiment_name = "training_spancat"
        run_name = "spancat_training"
        mlflow.set_experiment(experiment_name)

        # Begin mlflow run for logging
       with mlflow.start_run(run_name=run_name, experiment_id=None):
             config_path = "spancat/configs/run_config.cfg"
             cmd = [
                    "python", "-m", "spacy", "train",
                      config_path,
                      "-o", "spancat/models/thr_28/en/",
                      "--gpu-id", "-0",
                      "--paths.train", "data/train",
                      "--paths.dev", "data/dev",
                      "--nlp.lang", "en"
             ]
           subprocess.run(cmd)

**Comment by shadeMe** [comment, marked as answer]
Your config file looks like it's missing some important values. Please refer to the [documentation](https://github.com/explosion/spacy-loggers?tab=readme-ov-file#mlflowlogger) of the `MLflowLoger` for instructions on how to use it correctly.
